### ISSUE TITLE
a11y-no-background-images breaks Narrative when img tags have aria-hiddden

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-visua11y#50]
a11y-no-background-images hides img tags that don't have alt text. For Narrative, the controls become hidden as the height of `.narrative__slide-container` becomes 0 and `overflow-x: hidden` hides the controls preventing navigation between the items.

<img width="1196" alt="controls_hidden" src="https://user-images.githubusercontent.com/7045330/182633275-3a787d96-4b96-499f-be84-99d99ac376b0.png">

I suggest we include a component override in the [modifiers.less](https://github.com/cgkineo/adapt-visua11y/blob/db8cfc126f8b01f9c6d72ea34aa73832addcb877/less/modifiers.less#L65) when no background images are enabled.

**Comment by kirsty-hames** [comment]
I've gone through all core components with alt text removed and this doesn't impact elsewhere.

**Comment by joe-replin** [comment]
I've been made aware of a breaking bug in my project in relation to this. The breaking bug happens under these conditions:

- visua11y 'no background images' set to enabled
- graphic component set to required for in-view completion
- graphic component with no alt text (aria-hidden=true)

What results is the graphic component (as well as relative block, article & page) never achieving a status of complete. Any locking mechanisms in place never unlock and the learner is stuck in the course. 

Wondering if hiding the images instead of setting their display to none would be preferable here? That way, the expected space being taken up by graphics on the page is still there and any inview completions would be recognized?

`.a11y-no-background-images {
  img[aria-hidden=true] { **visibility: hidden;** }
}`
